frontend/shared/GithubBuildShaLink.test.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import GithubBuildShaLink from './GithubBuildShaLink';
+
+const defaultBuild = {
+  clonedCommitSha: 'cloned_sha',
+  requestedCommitSha: 'requested_sha'
+}
+const defaultSite = {
+    owner: 'repo_owner',
+    repository: 'repo_name'
+};
+const defaultProps = {...{ build: defaultBuild, site: defaultSite}}
```

Comment:
No need to spread an object. It can look like

```suggestion
const defaultProps = { build: defaultBuild, site: defaultSite }
```